hack/release.sh

```diff
@@ -59,25 +51,28 @@ if (( PUBLISH_RELEASE )); then
   echo "- Destination GCS: ${EVENTING_RELEASE_GCS}"
 fi
 
-# Build the release
+# Build the components
 
 all_yamls=()
 
-for yaml in "${!RELEASES[@]}"; do
-  config="${RELEASES[${yaml}]}"
+for yaml in "${!COMPONENTS[@]}"; do
+  config="${COMPONENTS[${yaml}]}"
   echo "Building Knative Eventing - ${config}"
   ko resolve ${KO_FLAGS} -f ${config}/ > ${yaml}
   tag_images_in_yaml ${yaml} ${EVENTING_RELEASE_GCR} ${TAG}
   all_yamls+=(${yaml})
 done
 
-for yaml in ${CLUSTERBUS_YAMLS[@]}; do
-  clusterbus_yaml=${yaml/-bus-/-clusterbus-}
-  config="${RELEASES[${yaml}]}"
-  echo "Building Knative Eventing - ${config} (${clusterbus_yaml})"
-  sed -e 's/^kind: Bus$/kind: ClusterBus/g' ${yaml} > ${clusterbus_yaml}
-  tag_images_in_yaml ${clusterbus_yaml} ${EVENTING_RELEASE_GCR} ${TAG}
-  all_yamls+=(${clusterbus_yaml})
+# Assemble the release
+for yaml in "${!RELEASES[@]}"; do
+  echo "Assembling Knative Eventing - ${yaml}"
+  echo -n "" > ${yaml}
+  for component in $(echo ${RELEASES[${yaml}]} | tr ";" "\n"); do
+    echo "---" >> ${yaml}
+    echo "# ${component}" >> ${yaml}
+    cat ${component} >> ${yaml}
+  done
+  all_yamls+=(${yaml})
```

Comment:
You missed tagging the images here (line 79 in the original code). This is a bug. :(

Reply:
@adrcunha is it necessary to run `tag_images_in_yaml` again? This step is only concatenating yamls generated above which were [tagged here](https://github.com/knative/eventing/pull/532/files/7aa9f7582369cad12e317c9d07e68910b771a254#diff-cede583a27c227d458ca67079957c8a2R62).

Reply:
Yes. In line 62 you were tagging the component yamls. Now you're tagging the release yamls. Because on line 88 you're publishing all yamls (components and releases).

Reply:
(sorry, this fell off my radar)